<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenOptionsWidgetClass</class>
 <widget class="QWidget" name="PenOptionsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PenOptionsWidget</string>
  </property>
  <widget class="QRadioButton" name="colorButton00">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>10</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="colorButton01">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>10</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="colorButton02">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>10</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="colorButton10">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>70</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="colorButton11">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>70</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="colorButton12">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>70</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="sizeSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>3</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
